yb-voyager/cmd/assessMigrationCommandBulk.go

```diff
@@ -336,13 +387,97 @@ func validateBulkAssessmentDirFlag() {
 	}
 }
 
-/*
-	TODO:
-		check if value valid or not,
-		expected/mandatory params are passed
-		strip any trailing spaces
-	func validateFleetConfigFilePath() {}
-*/
+var fleetConfigFileHeaderFields = []string{"dbtype", "hostname", "port", "service_name", "sid",
+	"tns_alias", "username", "password", "schema"}
+
+// TODO: verify/update this list
+var mandatoryHeaderFields = [][]string{
```

Comment:
Seems like a confusing way to implement this IMO. 
We should have just one list of mandatory field. Additionally, we can ensure that only one of (db_name,sid,tns_alias) is passed.

Reply:
done